tests/io/test_scans.py

```diff
@@ -2059,3 +2059,42 @@ def test_scan_create_scan_document_credentials_pass(api):
     for credential in credentials:
         getattr(api.scans, '_create_scan_document')(
             {'credentials': credential})
+
+
+@pytest.mark.vcr()
+def test_block_while_running(api):
+    '''
+    test _block_while_running with expected type.
+    '''
+    api.scans._block_while_running(scan_id=583, sleeper=5)
```

Comment:
Is an assertion not necessary here?

Reply:
In positive testcase we followed structure of existing testcase (`test_scan_create_scan_document_credentials_pass`) written just above this.